src/raftstore/store/peer_storage.rs

```diff
@@ -453,8 +453,13 @@ impl Storage for PeerStorage {
         self.initial_state()
     }
 
-    fn entries(&self, low: u64, high: u64, max_size: u64) -> raft::Result<Vec<Entry>> {
-        self.entries(low, high, max_size)
+    fn entries(
+        &self,
+        low: u64,
+        high: u64,
+        max_size: impl Into<Option<u64>>,
```

Comment:
IMO the `impl Into` is not worth it, just use an `Option` and the callers can write `Some` if they need too. You can keep it if you really like it.

Reply:
I think write pure number is quite handy. Besides, it's a method of the trait Storage, I don't think I can change the definition in TiKV.